openshift/kustomize/tekton/base/tasks/git-conditions.yaml

```diff
@@ -53,7 +53,15 @@ spec:
         git config --global --add safe.directory '*'
         git fetch origin master
         git fetch origin dev
+
+        # Delete head file if it exists
```

Comment:
This section helps resolve some of the git issues.  For some reason we cannot access these git files through a different container.  It appears to me that these containers are running as root somehow, which should be impossible in Openshift.